sdk/batch/Microsoft.Azure.Batch/src/Microsoft.Azure.Batch.csproj

```diff
@@ -43,5 +43,6 @@
   </ItemGroup>
   <PropertyGroup>
     <CodeAnalysisRuleSet>AzureBatch.ruleset</CodeAnalysisRuleSet>
+    <RunCodeAnalysis>false</RunCodeAnalysis>
```

Comment:
Don't we want to run code analysis? Otherwise what's the point of the ruleset?

Reply:
Thanks for pointing that out. I was just tying it out, but it still didn't stop code analysis on batch. I will removed it.